src/components/product-option/product-option-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { products } from '@wix/stores';
+
+export function getChoiceValue(
+    option: products.ProductOption | undefined,
+    choice: products.Choice | undefined
+): string | undefined {
+    if (option !== undefined && choice !== undefined) {
```

Comment:
```
if (option === undefined || choice === undefined) {
  return;
}
return option.optionType === products.OptionType.color ? choice.description : choice.value
```
?

and let's write comment why we do this

Reply:
fixed